File: features/merchant/review/src/main/java/com/tokopedia/review/feature/inbox/buyerreview/domain/model/inboxdetail/ProductDataDomain.java
package com.tokopedia.review.feature.inbox.buyerreview.domain.model.inboxdetail;

/**
 * @author by nisie on 8/23/17.
 */

public class ProductDataDomain {
    private int productId;
    private String productName;
    private String productImageUrl;
    private String productPageUrl;
    private int shopId;
    private int productStatus;

    public ProductDataDomain(int productId, String productName,
                             String productImageUrl, String productPageUrl, int shopId, int productStatus) {
        this.productId = productId;
        this.productName = productName;
        this.productImageUrl = productImageUrl;
        this.productPageUrl = productPageUrl;
        this.shopId = shopId;
        this.productStatus = productStatus;
    }

    public int getProductId() {
        return productId;
    }

    public String getProductName() {
        return productName;
    }

    public String getProductImageUrl() {
        return productImageUrl;
    }

    public String getProductPageUrl() {
        return productPageUrl;
    }

    public int getShopId() {
        return shopId;
    }

    public int getProductStatus() {
        return productStatus;
    }
}

File: features/merchant/review_inbox/src/main/java/com/tokopedia/review/feature/inbox/buyerreview/domain/model/inboxdetail/ProductDataDomain.kt
package com.tokopedia.review.feature.inbox.buyerreview.domain.model.inboxdetail

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author by nisie on 8/23/17.
 */
class ProductDataDomain constructor(
    @SerializedName("productID") @Expose val productId: String,
    @SerializedName("productName") @Expose val productName: String,
    @SerializedName("productImageURL") @Expose val productImageUrl: String,
    @SerializedName("productStatus") @Expose val productStatus: Int
)
